components/src/core/Drawer/DrawerContext.ts

```diff
@@ -0,0 +1,9 @@
+import { createContext, useContext } from 'react';
+
+type DrawerContextType = {
+    isOpen: boolean;
+};
+
+export const DrawerContext = createContext<DrawerContextType>({ isOpen: true });
```

Comment:
Is there a rationale behind defaulting to `open: true` versus `open: false` or `null`? I don't think it will have an impact since we're only using it internally and we initialize it with a value, but I was just curious.

Reply:
>Is there a rationale behind defaulting to open: true versus open: false or null? 

There is -- the DrawerNavGroup is designed to be used on its own, so we need to provide some initial value to display both the title and subtitle